docs/source/how_tos/installation.rst

```diff
@@ -19,7 +19,7 @@ Steps to make a dsgrid Conda environment:
 
 .. code-block:: bash
 
-    conda create -n dsgrid python=3.10
```

Comment:
Is there an advantage to this over letting our setup.py control the requirements?

Reply:
On some environments (including, yes, Windows) it's good to let conda handle the very-compiled packages like numpy and pandas. erdantic seemed to fall in this category as well.

Reply:
I see why it's likely preferable to keep this simpler.